### ISSUE TITLE
How to use cx_Freeze if the file packaged is not a .pyc file, but a. pyd file?

### ISSUE THREAD

**Issue by zhenzi0322** [marcelotduarte/cx_Freeze#1911]
Are there any parameters that can be used directly

![image](https://github.com/marcelotduarte/cx_Freeze/assets/34839719/a633cad9-4311-47bd-a6f0-6301ad73cb74)

**Comment by marcelotduarte** [comment]
I do not understand what you want. Maybe the answer is in the [documentation](https://cx-freeze.readthedocs.io/en/latest/setup_script.html#build-exe).

.py: usually the input source code you wrote.
.pyc: is the compiled bytecode. If you import a module, Python will build a *.pyc file that contains the bytecode to make it easier and faster later.
.pyd: It's basically a Windows dll file.

**Comment by zhenzi0322** [comment]
Can we output pyd files

**Comment by marcelotduarte** [comment]
Yes, using Cython. If you have a module .py (or .pyx) you can make .pyd with Cython, then use cx_Freeze to make the executable, the .pyd is automatically chosen.

**Comment by zhenzi0322** [comment]
Using cx_ Freeze Can't you directly create output for pyd files?

**Comment by marcelotduarte** [comment]
No. But, the creation of .py to .pyd and the build can be made in the setup.py used to build the executable.
To convert a .py to C and build a .pyd (.dll) is a bit complex, so exist Cython and others. I used Cython.

**Comment by zhenzi0322** [comment]
Ok.
